packages/truffle-box/lib/utils/unbox.js

```diff
@@ -81,13 +76,45 @@ function fetchRepository(url, dir) {
   });
 }
 
-function copyTempIntoDestination(tmpDir, destination) {
-  return new Promise(function(accept, reject) {
-    fs.copy(tmpDir, destination, function(err) {
-      if (err) return reject(err);
-      accept();
-    });
-  });
+async function copyTempIntoDestination(tmpDir, destination) {
+    const currentContent = fs.readdirSync(destination);
+      if (currentContent.length > 1) {
+        const tmpContent = fs.readdirSync(tmpDir);
+        for (let file of tmpContent) {
+           if (currentContent.includes(file)) {
+             var overwriting = [
+  		{
+    		type: 'confirm',
+    		name: 'overwrite',
+    		message: `Overwrite ${file}?`,
+    		default: false
+  		}]
+             await inquirer.prompt(overwriting)
+               .then(async answer => {
+                 if (answer.overwrite) {
+                   try {
+                     await fs.remove(file);
```

Comment:
I'm not sure we need to manually remove the file, it looks like copy overwrites the file if it already exists.  This might mean we can get rid of more logic in this method to clean it up.  Or maybe I'm missing something.